src/utils/getErrorView.tsx

```diff
@@ -0,0 +1,19 @@
+import React, { useState } from "react";
```

Comment:
why do you move it in the utils folder? id doesnt look as utility, its fullf-fledged component

Reply:
it was the my mistake